Content.Shared/Ensnaring/Components/EnsnaringComponent.cs

```diff
@@ -64,6 +64,10 @@ public sealed partial class EnsnaringComponent : Component
     [DataField("canMoveBreakout")]
     public bool CanMoveBreakout;
 
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField]
```

Comment:
```suggestion
    [DataField]
```
ReadWrite is the default now, so you don't need this attribute. You can also remove it from the other datafields above. Change them to `[DataField]` as well. The name gets autoconverted to camelCase, but old files still do this manually. And add a summary to your new datafield.